akamai/src/test/java/nl/nn/adapterframework/extensions/akamai/NetStorageSenderTest.java

```diff
@@ -0,0 +1,116 @@
+/*
+   Copyright 2020 WeAreFrank.nl
```

Comment:
Ik doe in voorkomende gevallen: Copyright 2020 WeAreFrank!, dus niet .nl, maar met uitroepteken.